docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,468 +1,134 @@
-  Teleport's RDP client supports only secure algorithms
-  for making TLS connections, so we have to configure our Domain Controller 
-  to support those cipher suites as well.
-  This step is only *necessary* for Windows Server 2012 R2 Domain Controller as it does not support
-  secure algorithms by default. If it does not apply to you, you can skip this step and go to the [next step](#step-57-export-your-ldap-ca-certificate).
-</Admonition>
-
-In this step we'll create a new certificate template that uses elliptic curve cryptography, and then configure our GPO
-to use the newly created template to issue certificates used for Remote Desktop connections.
-
-### Create a certificate template
+If you already have Active Directory installed, skip to the next step. Otherwise,
+copy and paste the first command provided into a Windows PowerShell window. If
+you aren't already running AD Certificate services, copy and paste the second
+command after the first one completes and the server restarts:
 
-In this section, we will create a certificate template that uses elliptic curve P-384 and uses SHA384 as the signature algorithm.
+![Install Active Directory](../../img/desktop-access/install-active-directory.png)
 
+Once the server is restarted from one or both command executions, click **NEXT**.
 
-1. Open the Microsoft Management Console (MMC)
-
-```text
-Start > Control Panel > Administrative Tools > Certificate Authority
-```
-
-2. Open your CA computer and right-click on `Certificate Templates`, then select `Manage`.
-3. Find the `Computer` template on the list, right-click on it, then select `Duplicate Template`.
-4. In the `Compatibility` tab change `Certification Authority` to `Windows Server 2012 R2` and click `OK`.
-5. In the same tab change `Certificate recipient` to `Windows Server 2012 R2` and click `OK`.
-6. Go to the `General` tab and change `Template display name` to `RemoteDesktopAccess`. Make sure `Template name` is also `RemoteDesktopAccess`.
-7. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
-8. Next, in the `Extensions` tab select `Application Polices` and click the `Edit` button.
-9. Remove all entries from the list.
-10. Go to the `Security` tab, select `Domain Controllers` and give the group `Read` and `Enroll` permissions.
-11. Finally, create a template by clicking `OK`.
-12. Go back to the Certificate Authority window and right-click on `Certificate Templates`. Then:
-
-```text
-New > Certificate Template to Issue
-```
-Select `RemoteDesktopAccess` and click `OK`.
-
-### Update GPO to use a new certificate template
+Copy and paste the provided command into a Windows PowerShell window to download
+and run the 'configure Active Directory' script:
 
-In the group policy editor for `Teleport Access Policy`, select:
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
```

Comment:
I think it's probably fine, but technically this screenshot shows a config v3 which is not supported on Teleport 10. Up to you whether you want to go through the effort to get a v10 screenshot.

Reply:
I think it's fine. Folks can't copy/paste from an image after all.